debian/control

```diff
@@ -4,6 +4,6 @@ Build-Depends:
     dh-virtualenv,
 
 Package: clusterman
-Depends: python3.6, ${misc:Depends}, ${python:Depends}, ${shlibs:Depends}
+Depends: python3.7, ${misc:Depends}, ${python:Depends}, ${shlibs:Depends}
```

Comment:
we depended on py36, but we tested py37

Reply:
I did notice this a while ago too that we seem to be expecting 3.6 as base in some places and 3.7 in some others

Reply:
Totally makes sense, I think going with 3.7 here is reasonable to separate it out from the 3.7 -> 3.8 upgrade in a separate PR